src/recipients/recipient.ts

```diff
@@ -178,20 +178,7 @@ export function getRecipientVerificationStatus(
   e164NumberToAddress: E164NumberToAddressType,
   addressToVerificationStatus: AddressToVerificationStatus
 ): RecipientVerificationStatus {
-  if (!recipientHasNumber(recipient)) {
-    if (recipientHasAddress(recipient) && recipient.address in addressToVerificationStatus) {
-      switch (addressToVerificationStatus[recipient.address]) {
-        case true:
-          return RecipientVerificationStatus.VERIFIED
-        case false:
-          return RecipientVerificationStatus.UNVERIFIED
-        case undefined:
-          return RecipientVerificationStatus.UNKNOWN
-      }
-    } else {
-      return RecipientVerificationStatus.UNKNOWN
-    }
-  } else {
+  if (recipient.recipientType === RecipientType.PhoneNumber && recipientHasNumber(recipient)) {
```

Comment:
In an ideal world we would just use `recipient.recipientType === RecipientType.PhoneNumber ` to do a check like this, right? Like ideally `PhoneNumber` recipients *should* always have a phone number? If I'm understanding that right, maybe a TODO will be helpful here for future us to help explain why we have this double phone number check.

Reply:
this was mainly to make TS happy, I can add a comment.